packages/mc-scripts/extract-intl.js

```diff
@@ -225,32 +225,30 @@ const extractFromFile = async fileName => {
   // We usually do not build the translation-files because they are
   // handled by transifex and sync by the transifex CLI.
   if (shouldBuildTranslations) {
-    await Promise.all(
-      locales.map(async locale => {
-        const translationFileName = `${outputPath}/${locale}.json`;
+    locales.forEach(locale => {
```

Comment:
Ah so this was wrapping it into a promise needlessly?

Reply:
Yes, that was one of the new rules from the additional config

Reply:
I like it. Async behaviour in JS is "toxic"/breaking to the consumer so should be avoided.